extensions/sql-bindings/src/test/testUtils.ts

```diff
@@ -58,14 +60,35 @@ export class MockVscodeMssqlIExtension implements vscodeMssql.IExtension {
 	}
 }
 
+export class MockAzureFunctionExtensionAPI implements AzureFunctionsExtensionApi {
```

Comment:
Do you need to do this? You're already using typemoq directly, is there a reason you couldn't just make the mock directly on the api interface?

Reply:
I tested this and found that if I don't then there would be cases where the createFunction API call would result in this error: `azureFunctionApi.createFunction is not a function`

Reply:
I tested this and found that if I don't then there would be cases where the createFunction API call would result in this error: `azureFunctionApi.createFunction is not a function`

Reply:
Did you set up the mock callback for that function?

Reply:
Ah, I believe that I didn't setup the initialization of TypeMoq but I tried this again, and should be working now.